status-im/status-react
# feat(wallet)_: Add beta info box in activity tab

fixes #20861

### Summary

This PR adds the beta info box in the activity tab with links to chain explorers.

<img width="250" src="https://github.com/user-attachments/assets/b596f126-c473-444d-bd66-a29e3a7b54a9" />

### Platforms

- Android
- iOS

### Steps to test

- Open Status
- Navigate to the Wallet tab
- Navigate to any of the account's activity
- Verify the beta info box is shown
- Verify that tapping the chain explorer name takes the user to the respective chain explorer

status: ready
